src/stories/ColorBar.stories.tsx

```diff
@@ -70,6 +80,7 @@ InvertColorMap.args = {
   invertColorMap: true,
 };
 
+export { Template as ColorMapTemplate };
```

Comment:
I now reuse the whole `Template` instead of only `TemplateProps`

Reply:
Nice, good call 👍